src/transformers/models/instructblip/processing_instructblip.py

```diff
@@ -57,6 +57,7 @@ def __init__(self, image_processor, tokenizer, qformer_tokenizer):
     def __call__(
         self,
         images: ImageInput = None,
+        videos: ImageInput = None,
```

Comment:
cc @molbap since we'd like to standardize multimodal processors, this one isn't making it easier 😅 

at some point we will have a `VideoTextToTextPipeline`, and we'll need to make sure they all have the same API.

See also the `ImageTextToTextPipeline` which is worked on at https://github.com/huggingface/transformers/pull/29572

Reply:
Sure! Re to normalizing processors, for models taking video inputs, `vivit`, `videomae`, `tvlt` have `videos: ImageInput` but `tvp` has `videos: Union[ImageInput, List[ImageInput], List[List[ImageInput]]],`. However `x_clip` reuses `videomae`'s processor. 

Overall `ImageInput` is defined as a type union of Images or list of images. Looks like in the future we might prefer supporting _at least_ list of list of images so a VideoInput defined as such could make sense, or an union of types as done in x_clip.